### PR TITLE
Update reports to only include active schools

### PR DESCRIPTION
**Pull request by iant**

### CODE REVIEW

File: app/models/school.rb [hunk 1]
```diff
@@ -186,6 +186,7 @@ class ProcessDataError < StandardError; end
   scope :inactive,            -> { where(active: false) }
   scope :deleted,             -> { inactive.where.not(removal_date: nil)}
   scope :archived,            -> { inactive.where(removal_date: nil)}
+  scope :active_and_archived, -> { where(removal_date: nil)}
```
Comment (urbanwide) [inline app/models/school.rb:189]: Just checking - is ok not not check the active flag here - and rely on the removal date being nil as the primary indicator of being active or archived?

**Review by urbanwide — APPROVED** [review]
Looks good @iant - couple of comments / questions. Thanks

File: app/models/school.rb (hunk 1 above)
Comment (ldodds) [reply app/models/school.rb:189]: agree, might be clearer to include active + archived more explicitly

File: app/models/data_source.rb [hunk 2]
```diff
@@ -47,6 +47,10 @@ def to_csv
     end
   end
 
+  def meters_from_active_schools
```
[the remark this answers is not in the document]
Comment (iant) [reply app/models/data_source.rb:50]: :+1: yep, sounds good to me.  have pushed an update.

File: app/models/procurement_route.rb [hunk 3]
```diff
@@ -24,7 +24,7 @@ class ProcurementRoute < ApplicationRecord
   def to_csv
     CSV.generate(headers: true) do |csv|
       csv << csv_headers
-      meters.each do |meter|
+      meters_from_active_schools.each do |meter|
```
[the remark this answers is not in the document]
Comment (iant) [reply app/models/procurement_route.rb:27]: :+1:

File: app/models/school.rb (hunk 1 above)
Comment (iant) [reply app/models/school.rb:189]: :+1:

File: app/models/school.rb [hunk 4]
```diff
@@ -184,8 +184,9 @@ class ProcessDataError < StandardError; end
   #are archived, with chance of returning if we receive funding
   scope :active,              -> { where(active: true) }
   scope :inactive,            -> { where(active: false) }
-  scope :deleted,             -> { inactive.where.not(removal_date: nil)}
-  scope :archived,            -> { inactive.where(removal_date: nil)}
+  scope :deleted,             -> { inactive.where.not(removal_date: nil) }
+  scope :archived,            -> { inactive.where(removal_date: nil) }
+  scope :active_and_archived, -> { find_by_sql(active.to_sql + ' UNION ALL ' + archived.to_sql) }
```
Comment (urbanwide) [inline app/models/school.rb:189]: Is this the same as writing active.or(archived) ? Just to simplify a bit if so...

**Pull request merged**